client/src/components/latestCampaigns/index.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Typography, Anchor } from 'antd';
+import Campaign from '../common/campaignCard';
+import './style.less';
+
+const { Link } = Anchor;
+const { Title, Text } = Typography;
+function latestCampaigns({ campaigns }) {
+  const toCampaignsPage = (e) => {
+    e.preventDefault();
+    // Navigate to /Campaigns
+  };
```

Comment:
Why this function? you have already navigated in the link below.

Reply:
but it will refresh the page , i figured maybe we could add route navigation or a react dom redirect component idk!!?

Reply:
I think if you remove the onCick option that will not happen.